tests/filecheck/dialects/irdl/cmath.irdl.mlir

```diff
@@ -0,0 +1,51 @@
+// RUN: xdsl-opt %s | xdsl-opt | filecheck %s
+
+builtin.module {
+  // CHECK-LABEL: irdl.dialect @cmath {
+  irdl.dialect @cmath {
+
+    // CHECK: irdl.type @complex {
+    // CHECK:   %{{.*}} = irdl.is f32
```

Comment:
Why no CHECK-NEXT?

Reply:
Yeah, I think I messed up the MLIR test as well when I wrote it ;(